### ISSUE TITLE
Centos7 not building properly / unable to download packages

### ISSUE THREAD

**Issue by adamtyoung** [geerlingguy/drupal-vm#1680]
### Issue Type

 - Bug Report / Support Request

### Your Environment

```
Vagrant 2.0.1
VirtualBox 5.2.6r120293
ansible 2.4.2.0
  config file = None
  configured module search path = [u'/Users/adamtyoung/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Library/Python/2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.10 (default, Jul 15 2017, 17:16:57) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)]
```

### Your OS

 - **macOS High Sierra**
 
### Full console output

https://gist.github.com/adamtyoung/8c562b562ef1ff6125b7885e812bf4be

### Summary

Vagrant up / provision step fails consistently at the 'PHP package installation' step on Centos 7. Can SSH into machine, running ```sudo service network start``` does *not* fix anything. Have tried with multiple Centos7 Vagrant images.

**Comment by geerlingguy** [comment]
It looks like you're unable to see any of the Remi hosts or mirrors used for the package installation:

```
Could not resolve host: rpms.remirepo.net; Unknown error
Could not resolve host: mirror.team-cymru.org; Unknown error
Could not resolve host: mirrors.tuna.tsinghua.edu.cn; Unknown error
...
```

Can you try running the provision again? Sometimes it's just a networking / Internet issue and it resolves itself after a few hours.

If it does the same thing again, then it could be your network is blocking access to those URLs for some reason, or you may be behind a VPN or restrictive proxy, maybe?

If you manually run `curl --head rpms.remirepo.net` both on your Mac directly, and then inside the VM (`vagrant ssh` to log in), what do you get? You should get a `200 OK` response.

**Comment by adamtyoung** [comment]
No--my network is fine--Mac returns 200 code. SSH'ing into Vagrant not gets ```curl: (6) Could not resolve host: rpms.remirepo.net; Unknown error```. I cannot even ping google from the vagrant box. Again, running network start doesn't help, and vagrant provision always gives the same error.

The problem doesn't exist if I use the ubuntu image, only with Centos 7.

**Comment by geerlingguy** [comment]
Also, are you using `geerlingguy/centos7` or some other CentOS 7 base box? My base box is tested and seems to be working fine (at least just a few min ago from home). Can't speak to other ones—often the 'official' boxes don't work very well IME.

**Comment by adamtyoung** [comment]
Yes, was using the ```geerlingguy/centos7``` box.

**Comment by djvilione** [comment]
Confirmed, I had the same issue. `sudo service network start` resolved it for me.
